dashboard/app/views/pd/regional_partner_contact_mailer/unmatched.html.haml

```diff
@@ -1,6 +1,6 @@
 %p
   The following person has requested to connect with their local Code.org Regional Partner
-  = link_to('through this form,', 'https://studio.code.org/pd/regional_partner_contact/new')
+  = link_to('through this form,', 'https://code.org/educate/professional-learning/contact-regional-partner')
```

Comment:
nit: we could use `CDO.code_org_url` like we do with the `matched` email..

Reply:
Yes, I copped out with the "left it the way it was" excuse. I actually have a [Honeybadger error](https://app.honeybadger.io/projects/3240/faults/60633192) that relates to that `CDO.code_org_url` -- I think I see the problem, but could use a quick consult if you have a minute. If you have a minute to help me understand that error, I can also fix this? 🤝